tests/conftest.py

```diff
@@ -9,6 +10,14 @@
 from app.config import RuntimeEnvironment
 
 
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Is there a reason for the autouse? I see a possible increase of safety. Yet we don’t need to set the database name if we don’t use the database.

```suggestion
@pytest.fixture(scope="session")
```